nixos/modules/services/misc/gitlab.nix

```diff
@@ -633,6 +684,19 @@ in {
     # Use postfix to send out mails.
     services.postfix.enable = mkDefault true;
 
+    # Enable Docker Registry, if GitLab-Container Registry is enabled
+    services.dockerRegistry = optionalAttrs cfg.registry.enable {
+      enable = cfg.registry.enable;
```

Comment:
this can be true, as the whole attrset is inside a `optionalAttrs cfg.registry.enable`.